nixos/modules/services/cluster/k3s/default.nix

```diff
@@ -724,7 +732,11 @@ in
               };
             };
           };
-
+          nginx = {
+            repo = "oci://registry-1.docker.io/bitnamicharts/nginx";
```

Comment:
@superherointj Did you tested this by chance?

Reply:
The URL no. I see URL errors as 404. I reviewed code wise. The usual tests are running fine for me.
Which kind of problem you are having? Tests run but you have some failure?

Reply:
No problem. Just tried to offload testing of k3s to you because I don't have any real world environment to test it.